src/test/ui/const-generics/raw-ptr-const-param.stderr

```diff
@@ -10,10 +10,10 @@ error[E0308]: mismatched types
   --> $DIR/raw-ptr-const-param.rs:7:38
    |
 LL |     let _: Const<{15 as *const _}> = Const::<{10 as *const _}>;
-   |                                      ^^^^^^^^^^^^^^^^^^^^^^^^^ expected `{pointer}`, found `{pointer}`
+   |                                      ^^^^^^^^^^^^^^^^^^^^^^^^^ expected `0x000000000000000f : *const u32`, found `0x000000000000000a : *const u32`
    |
-   = note: expected type `Const<{pointer}>`
-              found type `Const<{pointer}>`
+   = note: expected type `Const<0x000000000000000f : *const u32>`
```

Comment:
The space before the `:` seems a bit weird, but I'm not familiar with the conventions around raw pointers.

Reply:
I don't think there are any existing conventions.^^

For integers it seems like we are printing e.g. `42u32`, so having the type in some way makes sense. But I agree the space looks odd. For comparison:

`Const<0x000000000000000f : *const u32>`
`Const<0x000000000000000f: *const u32>`

Reply:
The space was not my first idea, but I added it after seeing the fallback const param at the very end of this function.
```rust
                    p!(write("{:?} : ", ct.val), print(ct.ty))
```
I can remove it if that's the consensus.

Reply:
Good point, let's stay consistent then.